### PR TITLE
[Live Bidding] In-House integration

### PR DESCRIPTION
**Pull request by vguerci**
JIRA: EE-1266

### CODE REVIEW

File: CriteoPublisherSdk/Tests/IntegrationTests/CR_ProfileIdFunctionalTests.m [hunk 1]
```diff
@@ -176,24 +177,22 @@ - (void)testStandaloneNative_GivenAnyPreviousIntegration_UseStandaloneProfileId
...
+  CRBid *bid = [self getBidWithAdUnit:adUnit];
 
-  [self prepareInHouseTest:adUnit];
+  NSDictionary *request = [self cdbRequest];
+  XCTAssertEqualObjects(request[@"profileId"], @(CR_IntegrationFallback));
 
-  [self.criteo getBidForAdUnit:adUnit];
-  [self waitForIdleState];
+  CRBannerView *bannerView = [[CRBannerView alloc] initWithAdUnit:adUnit criteo:self.criteo];
+  [bannerView loadAdWithBid:bid];  // declares integration
```
Comment (TurpIF) [inline CriteoPublisherSdk/Tests/IntegrationTests/CR_ProfileIdFunctionalTests.m:189]: Could you add test for the other format: interstitial and native ?

**Review by TurpIF — CHANGES_REQUESTED** [review]
Didn't you forget the declaration of the profile ID when using advanced native API ?

**Comment by vguerci** [comment]
> Didn't you forget the declaration of the profile ID when using advanced native API ?

Just adapted existing tests ¯\\\_(ツ)_/¯. I guess that was like that as I am not sure we support advanced native ads for in house, but anyway, adding a commit to fix this.

File: CriteoPublisherSdk/Tests/IntegrationTests/CR_ProfileIdFunctionalTests.m (hunk 1 above)
Comment (vguerci) [reply CriteoPublisherSdk/Tests/IntegrationTests/CR_ProfileIdFunctionalTests.m:185]: Done

**Comment by vguerci** [comment]
Fixed by amending while rebasing, [diff link](https://github.com/criteo/ios-publisher-sdk/compare/7b7b9bc85c5314d9f24ca338c7f836ea4445b9aa..70175a184b2e2ca1da0e1d15993a9155241c071f) for easing review.

**Review by TurpIF — APPROVED** [review]

**Comment by vguerci** [comment]
Thanks 👍

**Pull request merged**